src/rcc.rs

```diff
@@ -576,191 +543,26 @@ impl GetBusFreq for APB2 {
     }
 }
 
+use crate::pac::rcc::RegisterBlock as RccRB;
```

Comment:
I don't like this alias, it doesn't seem like just typing out `RegisterBlock` directly in a few places is that annoying, and saves some jumping around when reading the code (unless there are multiple `RegisterBlock`s in scope here, though in that case I'd prefer to just write out `rcc::RegisterBlock`

Reply:
Fixed.